Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.04 as builder
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+# Timezone data is needed during the installation of dependencies,
+# since cmake depends on the tzdata package. In an interactive terminal,
+# the user selects the timezone at installation time. Since this needs
+# to be a non-interactive terminal, we need to setup some sort of default.
+# The UTC one seemed the most suitable one.
+RUN echo 'tzdata tzdata/Areas select Etc' | debconf-set-selections; \
+    echo 'tzdata tzdata/Zones/Etc select UTC' | debconf-set-selections; \
+    apt-get update && apt-get upgrade -y && apt-get install --no-install-recommends -y \
+    cmake \
+    ninja-build \
+    gcc \
+    g++ \
+    maven \
+    flex \
+    bison \
+    libxml2-utils \
+    dpkg-dev
```

Comment:
What is `dpkg-dev` needed for?

Reply:
I don't think it is needed for anything, I think this list might've been copied from one of our packaging builds.

Reply:
We need `patch` that this depends on and pulls as a dependency - not `dpkg-dev`. I will update this now.